### PR TITLE
JSRS Migration

### PR DESCRIPTION
**Pull request by fdsgusmao**

### CODE REVIEW

**Comment by fdsgusmao** [comment]
@jpalioto review JSRS Migration please

**Comment by fdsgusmao** [comment]
### Goals verified (1 out of 1)

- Goal: RollDice
    - Roll dice
    - Roll 6-sided dice
    - Roll 2 dice
    - Roll2 6-sided dice

**NO ERRORS:** The migration tool successfully migrated this capsule, no extra changes were necessary.

**Review by kangawoo44 — APPROVED** [review]
LGTM

**Comment by fdsgusmao** [comment]
@kangawoo44 

- Warnings:0
- Errors:0
- Stage-1 Deprecations:0
- Stage-2 Deprecations:0

image

**Review by abapayeViv — APPROVED** [review]
👍

File: capsule.bxb [hunk 1]
```diff
@@ -4,13 +4,17 @@ capsule {
   version (0.1.1)
   targets {
     target (bixby-mobile-en-US)
-    target (bixby-mobile-ko-KR){enabled (false)}
+    target (bixby-mobile-ko-KR) {
+      enabled (false)
+    }
+  }
+  runtime-version (2) {
```
Comment (abapayeViv) [inline capsule.bxb:11]: Is there a reason we're using `runtime-version(2)` instead of a more recent one?

Comment (kangawoo44) [reply capsule.bxb:11]: probably not. @fdsgusmao can we update to the latest runtime version and run the tests again to see if there are any issues?

**Pull request merged**